src/web/components/SharingPermission/ParticipantsTable.stories.tsx

```diff
@@ -15,6 +15,7 @@ export const Default = Template.bind({});
 Default.args = {
```

Comment:
could we please add another story that has `hideSelectAllCheckbox`, to clearly show the difference? I found it a bit confusing at first since we are hiding one checkbox and showing another one elsewhere (for sharing permissions table)

Reply:
Good point, added